resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,322 @@
+  );
+
+  const creditRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Credits -'],
+        primaryLabel: 'Credits',
+      }),
+    [allGameHubs],
+  );
+
+  const technicalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Technical -'],
+        primaryLabel: 'Technical',
+      }),
+    [allGameHubs],
+  );
+
+  const protagonistRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Protagonist -'],
+        primaryLabel: 'Protagonist',
+      }),
+    [allGameHubs],
+  );
+
+  const settingRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Setting -'],
+        primaryLabel: 'Setting',
+      }),
+    [allGameHubs],
+  );
+
+  const regionalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Regional -'],
+        primaryLabel: 'Regional',
+      }),
+    [allGameHubs],
+  );
+
+  const raFeatureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Meta -', 'RANews -', 'Custom Awards -'],
+        hubTitleExcludes: ['Meta - Language'],
+        primaryLabel: 'Meta',
+        altLabels: ['RANews', 'Custom Awards'],
+      }),
+    [allGameHubs],
+  );
+
+  const formatRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Format -'],
+        primaryLabel: 'Format',
+      }),
+    [allGameHubs],
+  );
+
+  // Calculate used hub IDs from all other categories (excluding misc).
+  const usedHubIdsFromOtherCategories = useMemo(() => {
+    const allRows = [
+      ...creditRowElements,
+      ...developerRowElements,
+      ...featureRowElements,
+      ...formatRowElements,
+      ...genreRowElements,
+      ...hackOfRowElements,
+      ...languageRowElements,
+      ...perspectiveRowElements,
+      ...protagonistRowElements,
+      ...publisherRowElements,
+      ...raFeatureRowElements,
+      ...regionalRowElements,
+      ...settingRowElements,
+      ...technicalRowElements,
+      ...themeRowElements,
+    ];
+
+    return new Set(allRows.map((row) => row.hubId).filter((id): id is number => Boolean(id)));
+  }, [
+    creditRowElements,
+    developerRowElements,
+    featureRowElements,
+    formatRowElements,
+    genreRowElements,
+    hackOfRowElements,
+    languageRowElements,
+    perspectiveRowElements,
+    protagonistRowElements,
+    publisherRowElements,
+    raFeatureRowElements,
+    regionalRowElements,
+    settingRowElements,
+    technicalRowElements,
+    themeRowElements,
+  ]);
+
+  const miscRowElements = useMemo(
+    () =>
+      buildMiscRowElements(allGameHubs, usedHubIdsFromOtherCategories, {
+        keepPrefixFor: ['Fangames'],
```

Comment:
Consider keeping the "Clones" prefix:
http://localhost:64000/game2/13151
![image](https://github.com/user-attachments/assets/57a56f3d-2956-4abe-ade1-482efbe38731)

Reply:
I wonder if it would be easier to have a list of prefixes _to_ strip: `['Misc.', 'Central']`.

Reply:
Updated in latest so the "Clones" prefix will be kept.

> I wonder if it would be easier to have a list of prefixes to strip: ['Misc.', 'Central'].

I had a similar thought. Though, after going through a bunch of the Misc. hubs, I think the level of effort would be roughly the same.